### ISSUE TITLE
high cpu usage from thumbnailer when attaching PDF to evolution

### ISSUE THREAD

**Issue by Eliminateur** [linuxmint/xreader#350]
```
xreader 2.2.3
 * Distribution: mint 19.2 cinnamon
```
**Issue**
When attaching a PDF file to a new mail in Evolution(or using "send in email") the program hangs for ~15 seconds with process "xreader-thumbnailer" using one core to 100%, after that it attached the file and the program is responsive again.
Changing the default PDF handler does nothing, same process hanging one core.
If it has been sent before, then it attaches instantly next time

**Steps to reproduce**
select a PDF in file manager "send in email", as well as drag-n-drop on a mail composition window in evolution

**Expected behaviour**
Instant attachment

**Other information**

**Comment by jlrpalomar** [comment]
Exactly the same issue. Still open in Mint 19.3 Cinnamon

**Comment by icarter09** [comment]
Does this issue occur in other email clients as well?

**Comment by jlrpalomar** [comment]
@icarter09 no, but I think the other client I use (Thunderbird), doesn't create those real thumbnails of the document. On the other hand Nemo does this (show PDF thumbnails) and it doesn't take so much or use so much CPU
